projects/ci-cd.md

```diff
@@ -0,0 +1,130 @@
+| GitLab         | [Distributed Tracing](https://docs.gitlab.com/ee/operations/tracing.html)                                                                                   |                                                                                                    | * Tracing in GitLab pipelines                                 |
+| pytest         | [pyTest OpenTelemetry plugin](https://pypi.org/project/pytest-otel/)                                                                                        |                                                                                                    | * Tracing of Python test                                      |
+| Maven          | [Maven OpenTelemetry extension](https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/maven-extension/README.md)                           |                                                                                                    | * Tracing of Maven builds                                     |
+| Ansible        | [Ansible OpenTelemetry Callback Plugin](https://docs.ansible.com/ansible/latest/collections/community/general/opentelemetry_callback.html)                  |                                                                                                    | * Tracing in Ansible playbooks                                |
+| JUnit          | [JUnit to OpenTelemetry](https://github.com/mdelapenya/junit2otlp)                                                                                          |                                                                                                    | * Tracing of JUnit test results                               |
+| Eiffel         | [Eiffel CI/CD Event Protocol](https://eiffel-community.github.io/)                                                                                          |                                                                                                    | * Event protocol for CI/CD observability and interoperability |
+| CDEvents       | [CD Event Protocol](https://cdevents.dev/)                                                                                                                  |                                                                                                    | * Event protocol for CI/CD observability and interoperability |
+| Tekton         | [Tracing Tekton Enhancement Proposal](https://github.com/tektoncd/community/blob/main/teps/0124-distributed-tracing-for-tasks-and-pipelines.md)             |                                                                                                    | * Tekton Task and Pipeline Tracing                            |
+
+## Staffing / Help Wanted
+
+### Required staffing
+
+All stages will require a project lead, domain experts, and engineers at a minimum. 
+
+* @horovits Project Lead (Logz.io)
+* @adrielp (Liatrio)
+* @thisthat (Dynatrace, Keptn)
```

Comment:
I would like to be involved in this project as well. Super exciting to be bringing observability to the platforms that engineers use to gain better insights as to how value flows through the system!  Let's go!  🚀   

```suggestion
* @thisthat (Dynatrace, Keptn)
* @ericchapman80 (Liatrio)
```

Reply:
Added!